modules/tinymce/src/core/main/ts/util/Quirks.ts

```diff
@@ -448,7 +448,12 @@ const Quirks = (editor: Editor): Quirks => {
       // no point to exclude Ctrl+A, since normalization will still run after Ctrl will be unpressed
       // better exclude any key combinations with the modifiers to avoid double normalization
       // (also addresses TINY-1130)
-      if (!VK.modifierPressed(e)) {
+
+      // The next three lines address TINY-4550
+      const body = editor.getBody();
+      const rng = editor.selection.getRng();
+      const selectionIsAllContent = rng.startContainer === rng.endContainer && rng.startContainer === body && rng.startOffset === 0 && rng.endOffset === body.childNodes.length;
```

Comment:
Can we move this out to a separate helper function? That'll have 2 benefits: 1) it's more readable and clear what's going on and 2) it won't evaluate it when it's not needed (e.g. if a modifier key is pressed).